### PR TITLE
improving ui of login and register page

### PR DESCRIPTION
**Pull request by Vishal-raj-1**

### CODE REVIEW

File: webapp/styles/main.scss [hunk 1]
```diff
@@ -327,6 +331,16 @@ form {
...
+
+.submit-button{
+  background-color: #393171;
+  color: white;
 }
+
+.login-with-google{
+  color: #393171;
+  border: 1px solid black;
+  background: white;
```
Comment (mkstayalive) [inline webapp/styles/main.scss:344]: don't use color names, always use #codes. and that to as scss variables. and all color definitions should go in `light.scss`. talk to naveen

File: webapp/styles/layout.scss [hunk 2]
```diff
@@ -158,13 +158,10 @@ body,
...
 
-.guest-card-holder {
-  padding-top: 200px;
-}
-
 .guest-card {
-  max-width: 400px;
+  max-width: 800px;
   width: 100%;
+  padding: 23px;
```
Comment (mkstayalive) [inline webapp/styles/layout.scss:164]: use bootstrap shorthands for padding and margin. like p-3 or p-4 classes

File: webapp/pages/auth/register.tsx [hunk 3]
```diff
@@ -57,57 +57,74 @@ function Register(props: serverSideProps) {
...
-              setFieldValue={setFieldValue}
-              setFieldTouched={setFieldTouched}
-              name="clusterLocation"
-            />
-            <ButtonSubmit className="form-control" />
-          </Form>
-        )}
-      </Formik>
       {/* <div className="mt-3 text-center">
         <Button
```
Comment (mkstayalive) [inline webapp/pages/auth/register.tsx:129]: remove all commented code

File: webapp/styles/main.scss [hunk 4]
```diff
@@ -327,6 +331,16 @@ form {
...
+  font-size: 14px;
```
Comment (mkstayalive) [inline webapp/styles/main.scss:345]: use scss variables like $font-size-md

**Comment by Vishal-raj-1** [comment]
## Sign In Page:

signin

## Sign Up Page:

signup

**Review by naveenpaul — CHANGES_REQUESTED** [review]
1. Cluster select drop down is not matching design. Please test that it works after changing it to Material design.
2. "to continue to use Castled" text color should have #777777 according to the design
3. All the labels have a different font weight in the design. They're lighter. Try 400/normal. Also the font size is 10px for labels. Is it the same on the code too?
4. On the Sign-up screen, at the bottom of the box, create a link - "Have an account? Sign In". This is not there in the design. But its good to have a sign-in link. On the Login screen, you have the "Create Account" link. Same design and placement.

File: webapp/styles/main.scss [hunk 5]
```diff
@@ -101,6 +101,10 @@ body {
   height: 30px;
 }
 
+.bnt-red {
+  color: red !important;
```
Comment (mkstayalive) [inline webapp/styles/main.scss:105]: use color codes

Comment (mkstayalive) [reply webapp/styles/main.scss:105]: also, color definitions should go in light.scss

**Comment by franklinio** [comment]
@naveenpaul  Wondering if the dark castled logo goes well on the signin page?

**Comment by mkstayalive** [comment]
@franklinio at the moment we have only light mode. some point later we will support dark mode as well. i think the dark logo will then look good.

**Pull request closed without merging**